ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activityCompositeAnswer.component.ts

```diff
@@ -109,11 +120,14 @@ export class ActivityCompositeAnswer implements OnChanges {
         }
         if (this.shouldSetPlaceholderToBeQuestionText(childQuestionBlock)) {
             newQuestionBlock.placeholder = newQuestionBlock.question;
-            newQuestionBlock.question = null;
+            newQuestionBlock.question = '';
             if (childQuestionBlock.isRequired) {
                 newQuestionBlock.placeholder += ' *';
             }
         }
+        if (this.block.childOrientation === ChildOrientation.Horizontal) {
+            newQuestionBlock.question = '';
+        }
```

Comment:
This looks correct. Don't think we ever want to show the question prompt in horizontal.